examples/src/main/scala/org/apache/spark/examples/streaming/StatefulNetworkWordCount.scala

```diff
@@ -44,18 +44,6 @@ object StatefulNetworkWordCount {
 
     StreamingExamples.setStreamingLogLevels()
 
-    val updateFunc = (values: Seq[Int], state: Option[Int]) => {
```

Comment:
Since `updateStateByKey` isn't removed, maybe it's better to keep this example and create a new file for `trackStateByKey`.